fields/field.metakeys.php

```diff
@@ -279,7 +279,7 @@ public function checkPostFieldData($data, &$message, $entry_id = null) {
 			}
 
 			// Return if it's allowed to be empty (and is empty)
-			if(empty($data[0]['value'])) return self::__OK__;
+			if(General::strlen($data[0]['value']) == 0) return self::__OK__;
```

Comment:
I tried this out using XML Importer and it seems that this fails if `$data` is empty. Using 

```php
	if(isset($data[0]['value']) && General::strlen($data[0]['value']) == 0) return self::__OK__;
```

works.